test/parallel/parallel.status

```diff
@@ -36,3 +36,34 @@ test-debug-signal-cluster                : PASS, FLAKY
 
 #covered by https://github.com/nodejs/node/issues/8271
 test-child-process-fork-dgram            : PASS, FLAKY
+
+[$jsEngine==chakracore]
```

Comment:
@kunalspathak updated. I moved the `buffer` tests to a x64-only subgroup. The list of tests on your comment above is missing a few tests that I included in this PR (e.g. `parallel/test-force-repl-with-eval`, that I took from the failure that happened here: https://ci.nodejs.org/view/All/job/node-test-binary-windows/RUN_SUBSET=3,VS_VERSION=vs2015,label=win2008r2/4171/console).